src/ecs/fargate/datadog-ecs-fargate.ts

```diff
@@ -159,10 +159,12 @@ export class DatadogECSFargateTaskDefinition extends ecs.FargateTaskDefinition {
 
     // Log configuration on props
     if (this.datadogProps.logCollection!.isEnabled) {
-      if (props.logging !== undefined) {
-        log.debug("Overriding logging configuration for container: ", id);
+      if (props.logging === undefined) {
+        // Allow users to define their own logging configurations per container
```

Comment:
nit: got confused by this comment on first read. is it meant for the else block?